website/content/docs/configuration/identity-based-rate-limit.mdx

```diff
@@ -0,0 +1,117 @@
+application behavior and distributed denial of service (DDoS) attack, use [rate
+limit quotas](/vault/docs/configuration/create-lease-count-quota) and [lease
+count quotas](/vault/docs/configuration/create-rate-limit-quota).
+
+The rate limit quotas enforce API rate limiting using a [token
+bucket](https://en.wikipedia.org/wiki/Token_bucket) algorithm. For Vault
+Enterprise clusters, the rate limit quota supports a **`group_by`** option to
+define a group of requests based on the characteristic they have in common, and
+put them in the same bucket. 
+
+The available `group_by` modes are:
+
+- `ip` - groups requests by their source IP address (_Default_)
+
+- `none` - groups together all requests that match the rate limit quota rule
+
+- `entity_then_ip` - groups requests by their entity ID for authenticated
+  requests that carry one, or by their IP for unauthenticated requests (or
+  requests whose authentication is not connected to an entity)
+
+- `entity_then_none` - groups requests by their entity ID when available, but
+  the rest is all grouped together (for example, unauthenticated requests, and
+  requests with authentication that is not connected to an entity)
+
+The `group_by` option with `entity_then_ip` or `entity_then_none` mode allows
+you to set a secondary rate limit (**`secondary_rate`**). This rate limit applies to
+the requests that fall under the `ip` or `none` groupings, while the
+authenticated requests that contain an entity ID are subject to the primary rate
+limit set by the `rate` parameter.
+
+**Example:**
+
+The command below creates a rate limit quota with rate of 1,000 requests per
+second with `group_by` mode is `entity_then_none`, and the secondary rate is
+2,000 requests per second.
+
+```shell-session
+$ vault write sys/quotas/rate-limit/my-rate \
+    rate=1000 \
+    group_by=entity_then_none \
+    secondary_rate=2000     
+```
+
+The `entity_then_none` or `entity_then_ip` mode groups requests based on their
+attached entity. This helps when your organization have:
+
+- many workloads using the same IP
+- single workloads using many IPs which may scale up or down
+- dynamic IPs that change frequently
+
+The group by `none` option creates one bucket for all requests at the designated
+level (namespace, mount, or path) for that rate limit. For example, if your
+organization provides Vault as a service to your customers, you segregate the
+customers each into their own namespace. The default behavior of any rate limit
+set for the namespace creates a bucket per IP. If the desired behavior is to set
+a collective rate limit for all entities and workloads coming into the
+namespace, the `none` option can achieve that.
+
+
+![Diagram indicating possible user paths](/img/resource-quotas/group-by_light.png#light-theme-only)
+![Diagram indicating possible user paths](/img/resource-quotas/group-by_dark.png#dark-theme-only)
+
+
+## Rate limit quota precedence
```

Comment:
This section isn't really specific to the new group_by modes. Maybe its a better fit for the more general RLQ page then? The only difference in adding it to the CE page is that the section talking about `inheritable` and namespaces could get an asterisk as that's an ent-only thing